discover/reference/articles/knowledge-base-configuration.markdown

```diff
@@ -0,0 +1,683 @@
+# Knowledge Base Configuration [](id=knowledge-base-configuration)
+
+Here you'll find a list of the available system settings for each Knowledge Base
+application's configuration, what the corresponding portlet properties were if
+you're migrating from Liferay 6.2., and how to override configuration settings. 
+
+This reference doc is organized into the following sections:
+
+- Knowledge Base System Settings
+- Overriding Settings
+
+## Knowledge Base System Settings [](id=knowledge-base-system-settings)
+
+The next few sections show the available system settings for each Knowledge Base 
+application along with possible values. Note that any configuration changes
+you make in System Settings will set the default configuration for the 
+corresponding application across all sites. The default values can be
+overwritten by the individual application instance in your site through the 
+*Configuration* menu of the portlet.
+
+### Knowledge Base Article Configuration [](id=knowledge-base-article-configuration)
+
```

Comment:
Even though this is reference documentation, you should still introduce the section and the upcoming list of definitions.